keyform/models.py

```diff
@@ -5,6 +5,7 @@
 from django.conf import settings
 from django.utils.timezone import now
 from django.utils.translation import ugettext_lazy as _
+from django.forms.widgets import (CheckboxInput, Select, SelectMultiple,)
```

Comment:
This line is unnecessary since this file doesn't use any of these. Also parenthesis is weird in an import statement.